src/kosha/app/search.clj

```diff
@@ -3,13 +3,14 @@
             [kosha.app.util :as util]))
 
 
-(defn get-ragam
+(defn get-ragams
   "Retrieves 10 best matches for given ragam name."
-  [{:keys [params] :as request}]
-  (let [ragams (db-search/ragams (:q params) 10)]
+  [query]
+  (let [ragams (db-search/ragams query 10)]
```

Comment:
No, let's not paginate server side now. If we need pagination for UX reasons, we can utilize the pagination feature in `reactable`.